examples/confetti/src/main/scala/com/example/confetti/Confetti.scala

```diff
@@ -33,27 +34,36 @@ object Confetti extends IndigoDemo[Unit, Unit, Model, Unit]:
   def initialViewModel(startupData: Unit, model: Model): Outcome[Unit] =
     Outcome(())
 
-  def updateModel(context: FrameContext[Unit], model: Model): GlobalEvent => Outcome[Model] =
-    case FrameTick if context.mouse.leftMouseIsDown =>
```

Comment:
So the removal of this bit breaks the demo. It shows that your code works (red+green for left, blue+yellow for right), but the demo now only allows for clicks, not hold downs. Can we allow hold down of either button?